### ISSUE TITLE
[BITV]: Note regarding WebAuthn is not accessible

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42595]
There are contrast issue and screenreader issues (no announce regarding a note).

Note "Your browser does not support WebAuthn." is not accessible:

![Screenshot from 2024-01-05 13-23-10](https://github.com/nextcloud/server/assets/6078378/403cb152-bffd-4deb-bfd4-f04417963d1a)

Please create a same Note like in navigation bar settings:

![Screenshot from 2024-01-05 12-55-36](https://github.com/nextcloud/server/assets/6078378/31ebaf74-fb05-4888-8b0c-19ad48a87c8f)
